README.md

```diff
@@ -131,6 +131,18 @@ If you have any questions, want to report a bug, request a feature or you just w
 
 You can find more options for reaching us on the [Support page](https://superface.ai/support).
 
+## Public API
+
+If you decide to use some function from this package in your own code you definetly can but be aware that not every part of package is version controlled.
+
+Only function and API of entities below is under version control:
```

Comment:
```suggestion
Only functions and APIs of entities below are a part of the public API, and can be safely relied upon not to break between semver-compatible releases.

Using other parts of this package is at your own risk.
```

Reply:
what do you think @Jakub-Vacek? :)

Reply:
rationale - "you can use this package, but beware, there are dragons somewhere!" is not really the message IMHO, it's more that "if you want to use this package, there's A, B & C and everything else is dragons"

Reply:
I like it :) 
A, B & C are also dragons but we will try to keep them dragons😅